crates/containerd-shim-wasm/Cargo.toml

```diff
@@ -34,22 +34,44 @@ tokio = { version = "1.38.0", features = [ "full" ] }
 futures = { version = "0.3.30" }
 wasmparser = "0.209.1"
 tokio-stream = { version = "0.1" }
-prost-types = "0.12" # should match version in containerd-shim
+prost-types = "0.12"                                                      # should match version in containerd-shim
 sha256 = { workspace = true }
 
 # tracing
 tracing = { workspace = true, optional = true }
+tracing-subscriber = { version = "0.3", features = [
```

Comment:
**nit:** bump versions, also a comment about keeping these versions in sync would be nice.